package.json

```diff
@@ -83,6 +82,7 @@
     "karma-jasmine": "~1.1.0",
     "karma-jasmine-html-reporter": "^0.2.2",
     "karma-spec-reporter": "0.0.26",
+    "moment": "^2.24.0",
```

Comment:
Hmm... why is this just a dev dependency now?  \me reads on!

Reply:
Huh.  Is this only for that 1 `alerts when unscheduling a future published episode` test?  Could we just convert that to basic js Date() and get rid of the dev dependency entirely?